src/modules/AsyncInventory.js

```diff
@@ -23,7 +24,7 @@ const AsyncInventory = ({ component, onLoad, store, history, innerRef, ...props
     return (
         <RBACProvider appName="inventory">
             <Provider store={store}>
-                <Router history={history}>
+                <Router history={history} basename={getBaseName(window.location.pathname)}>
```

Comment:
To my knowledge: why do we need a router here?

Reply:
The details sometimes rely on router, like clicking on link in gen info and showing modal. However this was because we were not sharing router. Let me give it a try without the router (it would at least ease up the upgrate to react v 6)